.travis.yml

```diff
@@ -1,3 +1,5 @@
+sudo: false
```

Comment:
Switched to container-based configuration instead of full image with sudo.  The ubuntu 'precise' distro has an older python with a security hole - won't allow use of PIP or installation of several packages needed for code coverage uploads to coveralls.io.